components/test_raftstore/src/cluster.rs

```diff
@@ -788,7 +788,7 @@ impl<T: Simulator> Cluster<T> {
             sleep_ms(20);
         }
 
-        panic!("find no region for {}", hex::encode_upper(key));
+        panic!("find no region for {}", tikv_util::hex_encode_upper(key));
```

Comment:
this needs to redact.

Reply:
It seems that this is inside `test_...` crate. Is it necessary to redact log in tests?

Reply:
ok, make sense.